nss-day-cohort-51/nss-day-cohort-51.github.io
# jl-feature-footer-20 added NSS logo and NSS media icons. Footer is re…

…sponsive.

# Description - Footer

## Type of change - Added footer with NSS logo and links 


# Testing Instructions
fetch branch, serve, and make sure footer is centered and responsive.


ADDITION:
Added footer with NSS logo and links

# Checklist:
